hack/.spelling_failures

```diff
@@ -0,0 +1,10 @@
+swagger
```

Comment:
can we skip `docs/` as well?

Reply:
There may be some typos in docs as well.

Reply:
atleast the files under `docs/man` and `doc/user-guide` are autogenerated from .go files. So we have to avoid any autogenerated files under doc files.

Reply:
@mkumatag Make sense.

Are there any other folders that should be ignored as well? 
/cc @fejta @liggitt @ixdy

Reply:
the README for misspell says it doesn't work well for python or bash, so that includes all BUILD files (as you have here) plus all shell scripts (which aren't here), which is unfortunate.

also, a minor nit: maybe sort this file?

Reply:
> the README for misspell says it doesn't work well for python or bash,

@ixdy Yeah, I notice that warning. But that does not mean it won't work for python or bash. #59464 just contains some typos in bash or python that checked by misspell. IMO we can keep it for now, which does no harm currently. WDYT?